nemo/collections/tts/losses/fastpitchloss.py

```diff
@@ -120,6 +120,37 @@ def forward(self, pitch_predicted, pitch_tgt, len):
         return pitch_loss
 
 
+class EnergyLoss(Loss):
+    def __init__(self, loss_scale=0.1):
+        super().__init__()
+        self.loss_scale = loss_scale
+
+    @property
+    def input_types(self):
+        return {
+            "energy_predicted": NeuralType(('B', 'T'), RegressionValuesType()),
+            "energy_tgt": NeuralType(('B', 'T'), RegressionValuesType()),
+            "len": NeuralType(('B'), LengthsType()),
+        }
+
+    @property
+    def output_types(self):
+        return {
+            "loss": NeuralType(elements_type=LossType()),
+        }
+
+    @typecheck()
+    def forward(self, energy_predicted, energy_tgt, len):
+        dur_mask = mask_from_lens(len, max_len=energy_tgt.size(1))
+        ldiff = energy_tgt.size(1) - energy_predicted.size(1)
+        energy_predicted = F.pad(energy_predicted, (0, ldiff, 0, 0), value=0.0)
```

Comment:
In what situation are the predicted pitch/energy and target pitch/energy different lengths? Such logic could hide/mask bugs in the training code.